sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/util/IOHelperTest.java

```diff
@@ -0,0 +1,50 @@
+package com.mercedesbenz.sechub.wrapper.xray.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.mercedesbenz.sechub.wrapper.xray.XrayWrapperException;
+
+public class IOHelperTest {
+
+    final String testinputString = "test data";
+    InputStream inputStream;
+    IOHelper IOHelperToTest;
+
+    @BeforeEach
+    void beforeEach() {
+        inputStream = new ByteArrayInputStream(testinputString.getBytes());
+        IOHelperToTest = new IOHelper();
+    }
+
+    @Test
+    void readInputStreamAsString_return_content_as_string() throws XrayWrapperException {
+        /* execute */
+        String content = IOHelperToTest.readInputStreamAsString(inputStream);
+
+        /* test */
+        assertEquals(testinputString, content);
+    }
+
+    @Test
+    void readInputStreamAsString_throws_nullPointerException() {
+        /* execute + test */
+        assertThrows(NullPointerException.class, () -> IOHelperToTest.readInputStreamAsString(null));
```

Comment:
Either just no test, or we throw here an illegal argument exception. Testing a null exception is ... not very useful IMO